recipes/libtool/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,163 @@
+                autotools.make(args=["V=1"])
+                autotools.install()
+
+    def _test_autotools(self):
+        assert os.path.isdir(os.path.join(self._package_folder, "bin"))
+        assert os.path.isfile(os.path.join(self._package_folder, "include", "lib.h"))
+        assert os.path.isdir(os.path.join(self._package_folder, "lib"))
+
+        if not tools.cross_building(self):
+            self.run(os.path.join(self._package_folder, "bin", "test_package"), run_environment=True)
+
+    def _build_ltdl(self):
+        """ Build library using ltdl library """
+        with self._build_context():
+            if self.settings.compiler == "Visual Studio":
+                cmake = CMake(self, generator="NMake Makefiles")
+            else:
+                cmake = CMake(self)
+            cmake.configure(source_folder="ltdl")
+            cmake.build()
+
+    def _test_ltdl(self):
+        """ Test library using ltdl library"""
+        lib_suffix = {
+            "Linux": "so",
+            "FreeBSD": "so",
+            "Macos": "dylib",
+            "Windows": "dll",
+        }[str(self.settings.os)]
+
+        if not tools.cross_building(self):
+            bin_path = tools.unix_path(os.path.join("bin", "test_package"))
+            libdir = "bin" if self.settings.os == "Windows" else "lib"
+            lib_path = tools.unix_path(os.path.join(libdir, "liba.{}".format(lib_suffix)))
+            self.run("{} {}".format(bin_path, lib_path), run_environment=True, win_bash=tools.os_info.is_windows)
+
+    def _build_static_lib_in_shared(self):
+        """ Build shared library using libtool (while linking to a static library) """
+
+        # Copy static-in-shared directory to build folder
+        autotools_folder = os.path.join(self.build_folder, "sis")
+        shutil.copytree(os.path.join(self.source_folder, "sis"), autotools_folder)
+
+        install_prefix = os.path.join(autotools_folder, "prefix")
+
+        # Build static library using CMake
+        with self._build_context():
+            if self.settings.compiler == "Visual Studio":
+                cmake = CMake(self, generator="NMake Makefiles")
+            else:
+                cmake = CMake(self)
+            cmake.definitions["CMAKE_INSTALL_PREFIX"] = install_prefix
+            cmake.configure(source_folder=autotools_folder, build_folder=os.path.join(autotools_folder, "cmake_build"))
+            cmake.build()
+            cmake.install()
+
+        # Copy autotools directory to build folder
+        with tools.chdir(autotools_folder):
+            # Work around the fact that "used_special_vars" in conans/client/tools/win.py doesn't handle ACLOCAL_PATH
+            aclocal_path = "$ACLOCAL_PATH:" + self.deps_env_info.vars["ACLOCAL_PATH"][0].lower()
+            self.run("ACLOCAL_PATH={} autoreconf -ifv -Wall".format(aclocal_path), win_bash=tools.os_info.is_windows)
```

Comment:
```suggestion
            self.run("autoreconf -ifv -Wall", win_bash=tools.os_info.is_windows)
```
This simplification may be possible now that unix_path_package_info_legacy() is being used